src/Http/Controllers/NewsletterSubscriberController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers;
+
+use Blumilk\Meetup\Core\Http\Requests\Newsletter\NewsletterStoreRequest;
+use Blumilk\Meetup\Core\Http\Requests\Newsletter\NewsletterUpdateRequest;
+use Blumilk\Meetup\Core\Models\NewsletterSubscriber;
+use Blumilk\Meetup\Core\Services\NewsletterService;
+use Illuminate\View\View;
+
+class NewsletterSubscriberController extends Controller
+{
+    public function create(): View
+    {
+        return view("newsletter.index");
+    }
+
+    public function store(NewsletterStoreRequest $request, NewsletterService $service): View
```

Comment:
I think moving `NewsletterService` to the constructor will be better. You are using it 3 times.